<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>402</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="SearchTab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>GroupBox</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="searchSearchButton">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchLineEdit"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="filterButton">
               <property name="text">
                <string>Filter</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="rowCount">
             <number>10</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>120</number>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column>
             <property name="text">
              <string>List number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Log entry timestamp</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Log entry event</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Vector</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Reserved Area</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GraphTab">
       <attribute name="title">
        <string>Graph</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GraphMenu</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,5,5,1">
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <property name="horizontalSpacing">
            <number>4</number>
           </property>
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="graphSearchButton_2">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="graphLineEdit"/>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Add Node</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>ActualGraphArea</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="tabPosition">
             <enum>QTabWidget::East</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Graph Table</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QTableWidget" name="tableWidget">
                <property name="sortingEnabled">
                 <bool>true</bool>
                </property>
                <property name="rowCount">
                 <number>10</number>
                </property>
                <row>
                 <property name="text">
                  <string>Visibility</string>
                 </property>
                </row>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Timestamp</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Description</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Log Entry Reference</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Log Creator</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Event Type</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Icon Type</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Relathionships Table</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QTableWidget" name="RelationshipConfigTableWidget">
                <property name="rowCount">
                 <number>50</number>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>143</number>
                </attribute>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <row/>
                <column>
                 <property name="text">
                  <string/>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Relationship ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Parent</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Child</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Label</string>
                 </property>
                </column>
                <item row="0" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="1" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="2" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="3" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="4" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="5" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="6" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="7" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="8" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="9" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="10" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="11" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="12" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="13" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="14" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="15" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="16" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="17" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="18" column="0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="checkState">
                  <enum>Unchecked</enum>
                 </property>
                </item>
                <item row="19" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="20" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="21" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="22" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="23" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="24" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="25" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="26" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="27" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="28" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="29" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="30" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="31" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="32" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="33" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="34" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="35" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="36" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="37" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="38" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="39" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="40" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="41" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="42" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="43" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="44" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="45" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="46" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="47" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="48" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
                <item row="49" column="0">
                 <property name="checkState">
                  <enum>Checked</enum>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionAuto_Save"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind_2"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAuto_Save">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto-Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionClose_Exit">
   <property name="text">
    <string>Close/Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind_2">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
